exporter/googleservicecontrolexporter/exporter.go

```diff
@@ -0,0 +1,499 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googleservicecontrolexporter
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math"
+	"strings"
+	"sync"
+	"time"
+
+	scpb "cloud.google.com/go/servicecontrol/apiv1/servicecontrolpb"
+	"github.com/pborman/uuid"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/component/componentstatus"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/consumer/consumererror"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+	"google.golang.org/genproto/googleapis/api/distribution"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+const (
+	operationName = "OpenTelemetry Reported Metrics"
+	// OTel resource attribute that can override the static `consumer_project` setting from factory.go.
+	dynamicConsumerAttribute = "ucp.servicecontrol.consumer_id"
```

Comment:
Does this attribute make its way through to the consumer projects? I would like to remove the `ucp` namespace moving forward anyway - wondering if we can start here.

Reply:
Looks like it is removed if found. Feels safe to just rename.